src/main/java/com/getindata/connectors/http/sink/HttpSinkRequestEntry.java

```diff
@@ -0,0 +1,21 @@
+package com.getindata.connectors.http.sink;
+
+import lombok.EqualsAndHashCode;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+
+import java.io.Serializable;
+
+@RequiredArgsConstructor
+@EqualsAndHashCode
+public final class HttpSinkRequestEntry implements Serializable {
+  @NonNull
+  public final String method;
+  @NonNull
+  public final String contentType;
+  public final byte[] element;
```

Comment:
Could you separate each field with an empty line plz. 

Also add short java doc to every field and method. One sentence what each field represents.
Add short Javadoc on the top of the class.

I know that Table package has no such javadoc, but this is actually soethign that we could improve on.
I spend last months in an opensource project where Javadoc were something mandatory and believe me or not, it is helping later :)

P.S.
Yes I know about self documenting code, but never the less, short description will be something good.